src/TfvcMigrator/CommandLineException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TfvcMigrator;
+public sealed class CommandLineException : ApplicationException
```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.applicationexception#remarks

```suggestion
public sealed class CommandLineException : Exception
```